src/poprox_recommender/evaluation/metrics/rbo.py

```diff
@@ -1,24 +1,34 @@
-import numpy as np
+from lenskit.metrics import GeometricRankWeight
 
 from poprox_concepts.domain import CandidateSet
 
 
-def rank_biased_overlap(recs_list_a: CandidateSet, recs_list_b: CandidateSet, p: float = 0.9, k: int = 10) -> float:
+def rank_biased_overlap(recs_list_a: CandidateSet, recs_list_b: CandidateSet, p: float = 0.85, k: int = 10) -> float:
     """
     Computes the RBO metric defined in:
     Webber, William, Alistair Moffat, and Justin Zobel. "A similarity measure for indefinite rankings."
     ACM Transactions on Information Systems (TOIS) 28.4 (2010): 20.
 
     https://dl.acm.org/doi/10.1145/1852102.1852106
     """
-    summands = []
-    for d in range(1, k + 1):
+
+    if p == 0:
+        raise Exception("RBO: p=0")
+
+    rank_weight = GeometricRankWeight(p)
+    weights = [rank_weight.weight(d) for d in range(1, k + 1)]
```

Comment:
`weight` takes an array, so can just write:

```python
weights = rank_weight.weight(np.arange(1, k+1))
```

Creating a list here will create a list of vectors, likely won't do what is wanted.